MMM-PIR-Sensor.js

```diff
@@ -31,6 +32,25 @@ Module.register('MMM-PIR-Sensor',{
 			if (payload === false && this.config.powerSavingNotification === true){
 				this.sendNotification("SHOW_ALERT",{type:"notification", message:this.config.powerSavingMessage});
 			}
+
+			if(this.config.suspendResume) {
+				self = this;
+
+				// Suspend all modules
+				MM.getModules().enumerate((module) => {
+					if(!module.hidden) {
+						if(payload) { // User present
+							clearTimeout(self.suspendTimeout);
+							module.resume();
+						} else { // User not present
+							setTimeout(function() {
```

Comment:
Why set a separate timeout? Would it not be better to use the timeout defined in `note_helper.js` for reactivation, instead?